### ISSUE TITLE
Request for an active maintainer

### ISSUE THREAD

**Issue by lambdakilo** [JSONPath-Plus/JSONPath#191]
As the project has no active maintainer anymore, could you propose us a new one, @brettz9 ? JSONPath is a very important topic especially now that IETF is crafting the official standard for this. JSONPath-Plus is the second most active JSONPath tool in the world and the best tool for JavaScript right now. If there is no one I can definitely volunteer as the maintainer but only for this summer due to my university studies beginning in the fall.

https://github.com/JSONPath-Plus/JSONPath/blob/55e77286407db43282b4e5bdd232fb424cce9f11/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#LL6C1-L9C4

```md
---
<!--
**PLEASE NOTE: This project is not currently being very actively developed.**
-->
```

**Comment by akirataguchi115** [comment]
Good point. I agree that the next person should be involved in the cross-project comparison. To get more awareness for the next maintainer, could we pin this? Do you have some suggestions for an alternative title or is the current title descriptive enough?

**Comment by brettz9** [comment]
I think it's fine as is. Pinned.

**Comment by vid** [comment]
Hi all, I can offer to act as maintainer in terms of merging requests, following repo standards and trying to make things more clear, but I couldn't follow the developing jsonpath standards or make any promises about improving the repo code.

**Comment by brettz9** [comment]
> Hi all, I can offer to act as maintainer in terms of merging requests, following repo standards and trying to make things more clear, but I couldn't follow the developing jsonpath standards or make any promises about improving the repo code.

Thank you for the offer. I think I can manage the merging of requests at this point, but would welcome PRs to make things more clear as in other areas.

**Comment by 80avin** [comment]
A little announcement that I have joined this project as a maintainer now.

In the coming days, I will be working on implementing the IETF standard mentioned in https://github.com/JSONPath-Plus/JSONPath/issues/205 while retaining the path syntax supported by JSONPath-Plus.

This will also help in fixing many issues with the path parsing like https://github.com/JSONPath-Plus/JSONPath/issues/225 , https://github.com/JSONPath-Plus/JSONPath/issues/203 , https://github.com/JSONPath-Plus/JSONPath/issues/187 , etc.

**Comment by 80avin** [comment]
Closing this as I am maintaining it.
Any queries, let me know.
